cloudferrylib/os/compute/libvirt.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import os
+
+from xml.etree import ElementTree
+
+from cloudferrylib.utils import utils
+
+LOG = utils.get_log(__name__)
+
+nova_instances_path = "/var/lib/nova/instances/"
```

Comment:
I'd suggest resolving this in a separate commit. There's a lot of other places where we can get information from opentsack services config files, instead of hardcoded values or cloudferry configs.